Profiler_Test/ProfilerTest.cs

```diff
@@ -128,5 +128,25 @@ public void TestProfiling(string application, int[] expectedAssemblyIds)
             List<FileInfo> traces = RunProfiler(application);
             AssertNormalizedTraceFileEqualsReference(traces, expectedAssemblyIds);
         }
+
+        [Test]
+        public void TestAttachLog()
+        {
+            RunProfiler("ProfilerTestee.exe", arguments: "all", lightMode: true, bitness: Bitness.x86);
+            string[] lines = File.ReadAllLines(AttachLog);
+            string lastAttachLine = lines[lines.Length - 2];

```

Comment:
The `-2` is pretty magic number to me. Can we somehow make this more self-explanatory? E.g. by extracting to 2 into a variable with a self-explaining name? It's the second last line you check here, right? Maybe renaming `lastAttachLine` to `secondLastLogLine` already helps? Why is it not simply the first line?

Reply:
It's not the first line because the attach log is not newly created for the tests. If we used the first line of the file, we might get one that is pretty old (might even be days, weeks or longer, at least when testing on a dev machine). Since we want to test the log that we created with our current test run, we need to use the last and second last lines.

Reply:
Should we cleanup the files written by tests, say, in a `tearDown()`, to ensure there is no unexpected behavior due to data from previous tests or even test runs?

Reply:
:+1: